src/widget/wlibrarytableview.cpp

```diff
@@ -121,3 +122,7 @@ void WLibraryTableView::setTrackTableRowHeight(int rowHeight) {
     verticalHeader()->setDefaultSectionSize(math_max(
             rowHeight, fontHeightPx));
 }
+
+void WLibraryTableView::updateGeometries() {
```

Comment:
Hi,
You are right about librarytableview inherits from qtableview and the overwrite is not necessary however as per the earlier discussion with Daniel, we thought of overwriting the function as it would make code simple and easy to understand.